src/components/SidePanel.tsx

```diff
@@ -105,10 +104,8 @@ const SidePanelComponent: FC<SidePanelProps> = ({
   });
 
   return createPortal(
-    <aside
-      className={classnames("l-aside", className, {
-        "is-narrow": width === "narrow",
-        "is-wide": width === "wide",
+    <AppAside
+      className={classnames(className, {
```

Comment:
I think we need to still forward the `pinned` and `narrow` as well as the `wide` props to the AppAside. We can remove our custom classes for the three in here and just forward with those props.